pkg/crc/constants/constants.go

```diff
@@ -175,6 +176,26 @@ func EnsureBaseDirectoriesExist() error {
 	return nil
 }
 
+func initialiseAllDirectories(crcDir string) {
```

Comment:
This is not just about directories, as there are also sockets, and configuration files.